Viral_Load_Assay/src/org/labkey/viral_load_assay/Viral_Load_Manager.java

```diff
@@ -75,6 +83,32 @@ public JSONObject getDefaultAssayMetadata(JSONObject meta){
         plate.put("hidden", true);
         resultsMeta.put("plate", plate);
 
+        // This overrides extjs default decimalPrecision of 2, only where specified by metadata from the server. Makes input
+        // forms respect formatString on column metadata. Decimals will still round to two places if formatString not set.
+        // Consider: Move somewhere more central?
+        TableInfo resultTable = QueryService.get().getUserSchema(ctx.getUser(), ctx.getContainer(), SchemaKey.fromString("assay.Viral_Loads.Viral_Load")).getTable("Data");
```

Comment:
Carrying over my comment from the original PR, this is hard-coded with assumption that the assay design is named  Viral_Load. If that's true for all configurations that we care about, no worries.

Reply:
This is the only assay specific to this module, and the name is also hardcoded in xml config files.

Added the schema null check as discussed.